vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -147,9 +148,14 @@ vic_cesm_put_data()
 
                 // albedo: direct , visible
                 // CESM units: unitless
-                albedo = AreaFactor *
-                         (atmos->shortwave[NR] /
-                          (atmos->shortwave[NR] - energy.NetShortAtmos));
+                if (atmos->shortwave[NR] > 0.) {
+                    albedo = AreaFactor *
+                             (atmos->shortwave[NR] /
```

Comment:
You are right, this was wrong.  The logic has been corrected and comments were added.